packages/components/progress/src/composables/status.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://github.com/IDuxFE/idux/blob/main/LICENSE
+ */
+
+import { type ComputedRef, computed } from 'vue'
+
+import { type ConvertProgressSuccess, type ProgressProps, type ProgressStatus, progressStatus } from '../types'
+import { fullPercent } from '../util'
+
+export function useStatusClasses(
+  status: ComputedRef<ProgressStatus>,
+  mergedPrefixCls: ComputedRef<string>,
+): ComputedRef<string> {
+  const prefix = mergedPrefixCls.value
+  const statusClassMap: Record<ProgressStatus, string> = {
+    normal: `${prefix}-status-normal`,
+    exception: `${prefix}-status-exception`,
+    success: `${prefix}-status-success`,
+    active: `${prefix}-status-active`,
+  }
+
+  return computed(() => statusClassMap[status.value])
```

Comment:
```suggestion
  return computed(() => `${mergedPrefixCls.value}-status-${status.value}`)
```
感觉   `${mergedPrefixCls.value}-${status.value}` 是不是更好?